.github/workflows/check_protocol.yml

```diff
@@ -4,11 +4,12 @@ on:
   pull_request:
     types: [synchronize, opened, reopened, edited]  #en zie https://github.com/actions/runner/issues/980
     branches:
-     - main
+      - main
 
 jobs:
   build:
-    if: startsWith(github.head_ref, 'sfp-')
+    if: github.event.pull_request.draft == false
```

Comment:
Is this condition only needed for `sfp`-protocols?  Otherwise you need to use parentheses, as without parenthesis, the normal order of operations will be respected, so `and` will be performed first, and `or` afterwards.